docs/learn-polkadot-host.md

```diff
@@ -4,26 +4,41 @@ title: Polkadot Host (PH)
 sidebar_label: Polkadot Host (PH)
 ---
 
-The Polkadot Host is an important component of the Polkadot protocol. It consists of the networking, consensus and Wasm VM subsystems, and acts as the lower layers of the stack underlying the Polkadot runtime and the runtimes of the parachains. Loosely, the Polkadot Host can be thought of as synonymous with a virtual machine running the Polkadot Relay Chain (although as stated below, there is a clear separation between the chain logic itself, and the Polkadot Host on which it runs).
-
-The components of the Polkadot Host are:
-
-- The networking layer that facilitates network interactions.
+The architecture of Polkadot can be divided into two different parts,
+the Polkadot _runtime_ and the Polkadot _host_. The Polkadot runtime is the core
+state transition logic of the chain and can be upgraded over the course of time
+and without the need for a hard fork. In comparison, the Polkadot host is the
+environment in which the runtime executes and is expected to remain stable
+and mostly static over the lifetime of Polkadot.
+
+The Polkadot host interacts with the Polkadot runtime in limited, and well-specified
+ways. For this reason, implementation teams can build an alternative implementation
+of the Polkadot host while treating the Polkadot runtime as a black box. For more
+details of the interactions between the host and the runtime, please see the
+[specification](https://github.com/w3f/polkadot-spec/blob/master/runtime-environment-spec/polkadot_re_spec.pdf). 
+
+## Components of the Polkadot host
+
+- Networking components such as Libp2p that facilitates network interactions.
 - State storage and the storage trie along with the database layer.
-- Consensus engine.
+- Consensus engine for GRANDPA and BABE (some logic exists on the runtime).
```

Comment:
I'd clarify the "some logic..." part.

Reply:
I'll opt to remove it, going into details here would be out of place.